x-pack/plugins/reporting/common/job_utils.ts

```diff
@@ -5,7 +5,9 @@
  * 2.0.
  */
 
+import { isObject } from 'lodash';
+
 // TODO: Remove this code once everyone is using the new PDF format, then we can also remove the legacy
 // export type entirely
 export const isJobV2Params = ({ sharingData }: { sharingData: Record<string, unknown> }): boolean =>
-  Array.isArray(sharingData.locatorParams);
+  Array.isArray(sharingData.locatorParams) || isObject(sharingData.locatorParams);
```

Comment:
Would returning `sharingData.locatorParams != null` work more simply?

Reply:
Happy to go with your suggestion, I was trying to more strict in what values pass the test, but I don't think it adds much value in this case.